third_party/2and3/tabulate.pyi

```diff
@@ -0,0 +1,16 @@
+# Stub for tabulate: https://bitbucket.org/astanin/python-tabulate
+from typing import Any, Sequence
+
+
+def tabulate(
+    tabular_data: Sequence[Sequence[Any]],
```

Comment:
This should be at least `Iterable[Iterable[Any]]`. It seems that tabulate supports a few more formats, so maybe `Any` would be better for the moment.

Reply:
Done.